test/test_brain_atlas.py

```diff
@@ -70,6 +74,7 @@ def test_roi_indices(self):
                                extract_mode='vec',
                                rois=roi_list_rand_order)
             atlas.transform(self.X[:2])
+            # todo: why is that reordered?
```

Comment:
sure that is a new parameter. Since I was often confused for keeping the order of the rois which is in my opinion very important for handling the data array, I added this variable to be absolutely sure.